lumen_runtime/src/process/spawn/options.rs

```diff
@@ -85,6 +85,7 @@ impl Options {
             arity,
         });
         let (heap, heap_size) = self.sized_heap()?;
+        crate::system::io::puts(&format!("{:?}", heap));
```

Comment:
This looks like leftover debug printing and should probably be removed as printing the heap is expensive.